docker-compose.yaml

```
@@ -52,6 +52,13 @@ services:
             PIMCORE_INSTALL_MYSQL_PORT: 3306
             PIMCORE_INSTALL_MYSQL_HOST_SOCKET: db
             PIMCORE_INSTALL_MYSQL_DATABASE: pimcore
+            GOTENBERG_IP: "1.2.3.4"
```

```suggestion
            GOTENBERG_IP: "gotenberg"
```
Same here, we need to use the proper config, otherwise Gotenberg won't work when having the config in place using the env vars. 
Maybe we should also think about the naming, e.g. `GOTENBERG_HOST` is more specific? 🤔 We'd need to change this in pimcore/paas then as well.